metrics/converter.py

```diff
@@ -0,0 +1,57 @@
+import pandas
+import os
+import aif360.metrics
+import pandas as pd
+import aif360
+import numpy as np
+
+import itertools
+
+from aif360.metrics import BinaryLabelDatasetMetric, MDSSClassificationMetric
+from aif360.detectors import bias_scan
+
+from aif360.algorithms.preprocessing.optim_preproc_helpers.data_preproc_functions import load_preproc_data_compas
+
+from aif360.datasets import StructuredDataset
+from aif360.datasets import BinaryLabelDataset
+#from aif360.metrics import BinaryLabelDatasetMetric
+
+#from aif360.datasets import BankDataset
+
+import warnings
+warnings.filterwarnings("ignore", category=UserWarning)  # Suppress PyTorch warnings
+
+class Converter:
+    def __init__(self, df):
+        self.df = pandas.read_parquet(df)
+
+        # encode the 'race' column as binary 
+        race_map = {'White': 0, 'Black': 1, 'Hispanic': 1, 'Asian': 1,
```

Comment:
in the code below (my code), i considered the privileged group to be and unprivileged 0, so maybe just switch these values

Reply:
in lines 29-30